torch/csrc/jit/frontend/function_schema_parser.cpp

```diff
@@ -248,6 +248,7 @@ struct SchemaParser {
       case TypeKind::IntType:
       case TypeKind::BoolType:
       case TypeKind::FloatType:
+      case TypeKind::ComplexDoubleType:
         return parseSingleConstant(arg_type->kind());
```

Comment:
`parseSingleConstant` needs to be updated to handle complex numbers. I would either get rid of it and do it in a follow-up PR or add a test for it (and update `parseSingleConstant` and other code as needed).

Reply:
I'll just remove this change for the current PR and add it separately